constant.py

```diff
@@ -0,0 +1,49 @@
+# ENV FOLDERS
```

Comment:
It's a Python script as well then Add a License header as well + @author

Reply:
Done

Reply:
Done